package.json

```diff
@@ -252,6 +252,6 @@
     ]
   },
   "lint-staged": {
-    "*": "yarn eslint --ext .js,.jsx,.ts,.tsx --fix"
```

Comment:
No need for `ext` here, it would be ignored because `lint-staged` already passes a specific file. Instead, specify the glob to match staged files against.